api/nvidia/v1/clusterpolicy_types.go

```diff
@@ -1793,6 +1893,9 @@ func ImagePath(spec interface{}) (string, error) {
 	case *SandboxDevicePluginSpec:
 		config := spec.(*SandboxDevicePluginSpec)
 		return imagePath(config.Repository, config.Image, config.Version, "SANDBOX_DEVICE_PLUGIN_IMAGE")
+	case *DRADriverSpec:
```

Comment:
what does this do for dcgm-exporter. we will also need a flag to enable DRA metrics when dra driver is installed and let dcgm-exporter emit those metrics. Its not turned on by default.